estate/__manifest__.py

```diff
@@ -0,0 +1,12 @@
+{
+    'name': 'Real Estate',
+    'application': True,
+    'depends': ['base'],
+    'data': [
+        'security/ir.model.access.csv',
+        'views/estate_property_views.xml',
+        'views/estate_property_type_views.xml',
+        'views/estate_property_tag_views.xml',
+        'views/estate_menus.xml'
```

Comment:
Similary, in arrays, lists, sets and dictionaries, objects, we also include a comma `,` after the last entries: when a new entry will be added, that entry will not need to be impacted.